### PR TITLE
WebGLRenderer: Update docs and type definitions related to RenderTarget

### LINKED ISSUE #16084: WebGLRenderer: When its renderTarget is undefined, it simply doesn't work
### Description

When you do `renderer.setRenderTarget()` then attempt to `renderer.render()` ,
it doesn't work properly.
However, when you pass `null` instead of `undefined` , it works properly

[Documentation](https://threejs.org/docs/#api/en/renderers/WebGLRenderer.setRenderTarget) and [type definition](https://github.com/mrdoob/three.js/blob/7a7d6e222bcf7b7b6eff48cf0e9132b6a236ce3b/src/renderers/WebGLRenderer.d.ts#L343) says giving `undefined` to the `setRenderTarget` is not wrong usage.
(and type definition says we should not use `null` here)

Demo:

https://codepen.io/FMS-Cat/pen/LaKJvG?editors=0010

### Code / Stack Trace

This is a stack trace:

image

This part makes a mess I guess:

https://github.com/mrdoob/three.js/blob/7a7d6e222bcf7b7b6eff48cf0e9132b6a236ce3b/src/renderers/WebGLRenderer.js#L1208-L1218

### How should we fix this?

I completely recognize what went wrong, so I can propose these solutions:

- Accept both `undefined` and `null` as a render target
  - Automatically replace `undefined` render target with `null` , at beginning of `setRenderTarget`
  - Most preferable imo?
- Ban `undefined` render target
  - Also good to prevent confusion

Which one do you prefer? I'm not good at this context

### PR DESCRIPTION
**Pull request by 0b5vr**
will close #16084 

This commit bans giving `undefined` for `renderer.setRenderTarget` .
No code changes, just docs and type definitions.

### CODE REVIEW

File: src/renderers/WebGLRenderer.d.ts [hunk 1]
```diff
@@ -333,14 +333,24 @@ export class WebGLRenderer implements Renderer {
   ): void;
 
   /**
-   * @deprecated
+   * Returns the current RenderTarget if there are; returns null otherwise.
```
Comment (Mugen87) [inline src/renderers/WebGLRenderer.d.ts]: Maybe like so:

> Returns the current render target. If no render target is set, null is returned.

Comment (FMS-Cat) [reply src/renderers/WebGLRenderer.d.ts]: Thanks :pray:

Comment (FMS-Cat) [reply src/renderers/WebGLRenderer.d.ts]: ✔️

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
"Apply Mugen87-san's suggestion" 😁

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**